src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/abi/ppc64/sysv/SysVVaList.java

```diff
@@ -0,0 +1,289 @@
+/*
+ *  Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ *  DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ *  This code is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License version 2 only, as
+ *  published by the Free Software Foundation.  Oracle designates this
+ *  particular file as subject to the "Classpath" exception as provided
+ *  by Oracle in the LICENSE file that accompanied this code.
+ *
+ *  This code is distributed in the hope that it will be useful, but WITHOUT
+ *  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ *  FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ *  version 2 for more details (a copy is included in the LICENSE file that
+ *  accompanied this code).
+ *
+ *  You should have received a copy of the GNU General Public License version
+ *  2 along with this work; if not, write to the Free Software Foundation,
+ *  Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ *   Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ *  or visit www.oracle.com if you need additional information or have any
+ *  questions.
+ *
+ */
+
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2021, 2021 All Rights Reserved
+ * ===========================================================================
+ */
+
+package jdk.internal.foreign.abi.ppc64.sysv;
+
+import jdk.incubator.foreign.*;
+import jdk.incubator.foreign.CLinker.VaList;
+import jdk.internal.foreign.abi.SharedUtils;
+import jdk.internal.foreign.abi.SharedUtils.SimpleVaArg;
+
+import java.lang.invoke.VarHandle;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+import static jdk.internal.foreign.PlatformLayouts.SysVppc64le.C_POINTER;
+
+/**
+ * This file is copied from x86/windows (Windows/x86_64) as a placeholder for compilation
+ * as VaList on Linux/ppc64le at Java level is not yet implemented for the moment.
+ * The defintion VaList must map to the underlying struct of va_list defined on Linux/ppc64le
+ * which might be similar to Windows/x86_64. Thus, futher analysis on the struct is required
+ * to understand how the struct is laid out in memory (e.g. the type & size of each field in
+ * va_list) and how the registers are allocated for va_list.
+ */
+public class SysVVaList implements VaList {
+    static final Class<?> CARRIER = MemoryAddress.class;
+    private static final long VA_SLOT_SIZE_BYTES = 8;
+    private static final VarHandle VH_address = MemoryHandles.asAddressVarHandle(C_POINTER.varHandle(long.class));
+    private static final VaList EMPTY = new SharedUtils.EmptyVaList(MemoryAddress.NULL);
+
+    private MemorySegment segment;
```

Comment:
Why is `segment` not final like the other fields?

Reply:
There is another placeholder file (to be replaced in the future) specific to `VaList` which should remain unchanged as explained above.